src/Microsoft.Framework.Runtime/Infrastructure/JsonExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Framework.Runtime.Infrastructure
+{
+    internal static class JsonExtensions
+    {
+        public static string[] ValueAsStringArray(this JObject obj, string key)
```

Comment:
I think this can go away now as there's a new method that @Praburaj added that handles this for generic <T>